testsuite/tests/conftest.py

```diff
@@ -159,6 +159,8 @@ def pytest_report_header(config):
     threescale = settings["threescale"]["admin"]["url"]
     version = settings["threescale"]["version"]
     catalogsource = weakget(settings)["threescale"]["catalogsource"] % "UNKNOWN"
+    apicast_project = settings["threescale"]["gateway"]["OperatorApicast"]["openshift"]["project_name"]
```

Comment:
This might fail, I would use weakget just to be sure

Reply:
fixed.